providers/tests/system/amazon/aws/example_bedrock_retrieve_and_generate.py

```diff
@@ -442,9 +442,8 @@ def delete_opensearch_policies(collection_name: str):
     test_context = sys_test_context_task()
     env_id = test_context["ENV_ID"]
 
-    aoss_client = OpenSearchServerlessHook(aws_conn_id=None)
-    bedrock_agent_client = BedrockAgentHook(aws_conn_id=None)
-
+    aoss_client = OpenSearchServerlessHook()
+    bedrock_agent_client = BedrockAgentHook()
```

Comment:
This is a slight change in behavoiur -- if we don't specify this it will use `aws_default` as the connection.

I don't know if this matters or not

Reply:
BTW. IMHO this is actually the "correct" way of writing "example_dags/system_tests" - it should rely on whatever user has connection defined in their test environment - either "aws_default" (if  defined) or default boto3 mechanism -> then it will work as intended for the user when they are running it (and it's a good example on how it should be done - you should never set aws_conn_id = None deliberately in examples, so having None there is not a good example.

My guess is that @ferruzzi did the None thing to avoid some other failures or copied the behaviour from unit tests - but yeah. I would love to get confirmation if all my assumptions were correct :)

Reply:
> My guess is that @ferruzzi did the None thing to avoid some other failures or copied the behaviour from unit tests - but yeah.

I don't remember doing it to get around other issues.  I think maybe I added it in there to avoid hitting the db but clearly that was the wrong answer.  I just did a quick check and we're not doing `=None` in any other system tests, so as long as it's still passing (which Vincent already confirmed)  it's fine.

Reply:
Cool. Yeah. I did not see it in other system tests either, and since the SSM access was only in this bedrock test. I figured that must be somewhat connected :). Merging then :)